ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixClusterAggregator.java

```diff
@@ -39,28 +41,37 @@
   }
 
   /**
-   * Take a {@link List} of JSON string representation of {@link StatsWrapper} objects and perform cluster wide
+   * Take a {@link Map} of instance name to JSON string representation of {@link StatsWrapper} objects and perform cluster wide
    * aggregation with them.
-   * @param statsWrappersJSON a {@link List} of JSON string representation of {@link StatsWrapper} objects from the
+   * @param statsWrappersJSON a {@link Map} of instance name to JSON string representation of {@link StatsWrapper} objects from the
```

Comment:
Yes, this logging will be very useful during investigation. If not, we have no way to differentiate between values from different instances. This will be run as a helix task. One of the storage node will execute this on triggering.